Week01/Problem03/afilbrun_03.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul 12 13:32:50 2018
+
+@author: afilbrun
+"""
+
+# Problem 3.
+
+import math
+value = 600851475143;
+
+factors = [];
+prime_factors = [];
+
+for n in range(int(math.sqrt(value)),2,-1):
```

Comment:
Nice! Good identification of an easy way to save time by reversing the loop, and reducing the total number of values to check.